package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "engines": {
-    "node": "^14.15.0"
+    "node": ">=18.20.0"
```

Comment:
should this be 20 as the docker image is 20

Reply:
updated node version now